symplyphysics/laws/hydro/inner_friction_is_proportional_to_velocity_gradient.py

```diff
@@ -0,0 +1,58 @@
+from sympy import Eq, Derivative
+from symplyphysics import units, Quantity, Symbol, Function, print_expression, validate_input, validate_output
+
+# Description
+## In a steady (laminar) flow in a tube, the speed of fluid particles varies from its maximum value
+## (along the tube axis) to zero (near tube walls). Each layer decelerates the layer closer to the axis,
+## and accelerates the layer further from the axis. In between the fluid layers there exist forces of
+## inner friction.
+
+# Law: F = eta * dv/dy * S
+## F - forces of inner friction between fluid layers
+## eta - dynamic viscosity
+## v - speed of fluid flow
+## d/dy - spatial derivative in the direction perpendicular to velocity vector (i.e., a gradient)
+## S - area of fluid layers
```

Comment:
What is area of layer of fluid in a tube?
Usually this law is applied for a moving plate. I suggest to update description to make it clear how to properly apply it to some real problems, eg to to a plate on a layer of fluid.

Reply:
Now I believe it's better to rewrite this law in terms of shear stress (and have a separate definition of it), as it is usually presented in most sources related to fluid mechanics, plus it would simplify the law and remove the potential misunderstanding of what layer area is.

Reply:
I agree.